src/main/java/com/ning/http/client/AsyncHttpClientConfig.java

```diff
@@ -1043,6 +1051,11 @@ public Builder setMaxConnectionLifeTimeInMs(int maxConnectionLifeTimeInMs) {
            return this;
         }
 
+        public Builder setAcceptAnyCertificates(boolean acceptAnyCertificate) {
```

Comment:
Add javadoc as other setters?

Reply:
This is coming from the library directly, that's why.